From 755fef17815bf130f6b092b23a99d77bcf3963a1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Miroslav=20Such=C3=BD?= <msuchy@redhat.com>
Date: Wed, 13 May 2020 09:33:29 +0200
Subject: [PATCH] remove old transition postscriptlet

I think that after nine years, we can safely assume everyone done this migration.
---
 abrt.spec.in | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/abrt.spec.in b/abrt.spec.in
index 4c01fffe6..f8cebffe3 100644
--- a/abrt.spec.in
+++ b/abrt.spec.in
@@ -694,10 +694,6 @@ exit 0
 %systemd_post abrtd.service
 
 %post addon-ccpp
-# this is required for transition from 1.1.x to 2.x
-# because /cache/abrt-di/* was created under root with root:root
-# so 2.x fails when it tries to extract debuginfo there..
-chown -R abrt:abrt %{_localstatedir}/cache/abrt-di
 %systemd_post abrt-ccpp.service
 %systemd_post abrt-journal-core.service
 %journal_catalog_update
-- 
2.21.3

